### PR TITLE
Work Focus Penalty Reduction Augmentation

### PR DESCRIPTION
**Pull request by Nolshine**

### CODE REVIEW

File: src/Augmentation/Augmentation.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ interface IConstructorParams {
   programs?: string[];
 }
 
-function generateStatsDescription(mults: IMap<number>, programs?: string[], startingMoney?: number): JSX.Element {
+function generateStatsDescription(mults: IMap<number>, programs?: string[], startingMoney?: number, focusBonus?: number): JSX.Element {
```
Comment (hydroflame) [inline src/Augmentation/Augmentation.tsx:57]: Revert, use stats instead. Like RedPill or HiveMind

File: src/Augmentation/Augmentation.tsx [hunk 2]
```diff
@@ -390,6 +390,10 @@ export class Augmentation {
   // The Player/Person classes
   mults: IMap<number> = {};
 
+  // Focus penalty reduction given by this Augmentation.
+  // Isn't really a multiplier, so it's out here on its own.
+  focusBonus?: number;
```
Comment (hydroflame) [inline src/Augmentation/Augmentation.tsx:395]: Unused

Comment (Nolshine) [reply src/Augmentation/Augmentation.tsx]: oops

**Comment by hydroflame** [comment]
Added independelty, credits will still be given

**Pull request closed without merging**